# AI4Bio-Reading-List

This is a AI for Biology reading list maintained by the MBZUAI AI4Bio Group.

**Contents:**
- [1. Protein Level](#1-protein-level)
  * [1.1 Pilot Work](#11-pilot-work)
  * [1.2 Protein Structure Prediction](#12-protein-structure-prediction)
  * [1.3 Protein Representation Learning](#13-protein-representation-learning)
- [2. Protein Interaction Level](#2-protein-interaction-level)
- [3. Cell Level](#3-cell-level)
- [4. Others](#4-others)



## 1. Protein Level


### 1.1 Pilot Work
* [Highly accurate protein structure prediction with AlphaFold](https://www.nature.com/articles/s41586-021-03819-2). *Nature*. Jumper, J., Evans, R., Pritzel, A., Green, T., Figurnov, M., Ronneberger, O., ... & Hassabis, D. (2021).



### 1.2 Protein Structure Prediction

* [Accurate prediction of protein structures and interactions using a three-track neural network](https://www.science.org/doi/abs/10.1126/science.abj8754). *Science*. Baek, M., DiMaio, F., Anishchenko, I., Dauparas, J., Ovchinnikov, S., Lee, G. R., ... & Baker, D. (2021).

* [Language models of protein sequences at the scale of evolution enable accurate structure prediction](https://www.biorxiv.org/content/10.1101/2022.07.20.500902v1.full.pdf). *BioRxiv preprint*. Lin, Z., Akin, H., Rao, R., Hie, B., Zhu, Z., Lu, W., ... & Rives, A. (2022).

* [High-resolution de novo structure prediction from primary sequence](https://www.biorxiv.org/content/10.1101/2022.07.21.500999v1.full.pdf). *BioRxiv preprint*. Wu, R., Ding, F., Wang, R., Shen, R., Zhang, X., Luo, S., ... & Peng, J. (2022).

* [Helixfold-single: Msa-free protein structure prediction by using protein language model as an alternative](https://arxiv.org/pdf/2207.13921.pdf). *ArXiv preprint*. Fang, X., Wang, F., Liu, L., He, J., Lin, D., Xiang, Y., ... & Song, L. (2022). 

* [AmoebaContact and GDFold: a new pipeline for rapid prediction of protein structures](https://arxiv.org/pdf/1905.11640.pdf). *arXiv preprint arXiv:1905.11640*. Mao, W., Ding, W., & Gong, H. (2019).
* [Improved fragment sampling for ab initio protein structure prediction using deep neural networks](https://www.nature.com/articles/s42256-019-0075-7).*Nature Machine Intelligence*. Wang, T., Qiao, Y., Ding, W., Mao, W., Zhou, Y., & Gong, H. (2019). Improved fragment sampling for ab initio protein structure prediction using deep neural networks. Nature Machine Intelligence, 1(8), 347-355.
* [Improved protein structure prediction by deep learning irrespective of co-evolution information](https://www.nature.com/articles/s42256-021-00348-5). *Nature Machine Intelligence*. Xu, J., Mcpartlon, M., & Li, J. (2021). Improved protein structure prediction by deep learning irrespective of co-evolution information. Nature Machine Intelligence, 3(7), 601-609. 
* 





### 1.3 Protein Representation Learning

* [Biological structure and function emerge from scaling unsupervised learning to 250 million protein sequences](https://www.pnas.org/doi/epdf/10.1073/pnas.2016239118). *PNAS*. Rives, A., Meier, J., Sercu, T., Goyal, S., Lin, Z., Liu, J., ... & Fergus, R. (2021). 
* [Leveraging implicit knowledge in neural networks for functional dissection and engineering of proteins](https://www.nature.com/articles/s42256-019-0049-9).*Nature Machine Intelligence*. Upmeier zu Belzen, J., Bürgel, T., Holderbach, S., Bubeck, F., Adam, L., Gandor, C., ... & Eils, R. (2019). Leveraging implicit knowledge in neural networks for functional dissection and engineering of proteins. Nature Machine Intelligence, 1(5), 225-235.
* [Protein function prediction is improved by creating synthetic feature samples with generative adversarial networks](https://www.nature.com/articles/s42256-020-0222-1). *Nature Machine Intelligence*. Wan, C., & Jones, D. T. (2020). Protein function prediction is improved by creating synthetic feature samples with generative adversarial networks. Nature Machine Intelligence, 2(9), 540-550.
* [Protein function prediction for newly sequenced organisms](https://www.nature.com/articles/s42256-021-00419-7). *Nature Machine Intelligence*. Torres, M., Yang, H., Romero, A. E., & Paccanaro, A. (2021). Protein function prediction for newly sequenced organisms. Nature Machine Intelligence, 3(12), 1050-1060.
* [Expanding functional protein sequence spaces using generative adversarial networks](https://www.nature.com/articles/s42256-021-00310-5). *Nature Machine Intelligence*. Repecka, D., Jauniskis, V., Karpus, L., Rembeza, E., Rokaitis, I., Zrimec, J., ... & Zelezniak, A. (2021). Expanding functional protein sequence spaces using generative adversarial networks. Nature Machine Intelligence, 3(4), 324-333.
* [Transformer-based protein generation with regularized latent space optimization](https://www.nature.com/articles/s42256-022-00532-1). *Nature Machine Intelligence*. Castro, E., Godavarthi, A., Rubinfien, J., Givechian, K., Bhaskar, D., & Krishnaswamy, S. (2022). Transformer-based protein generation with regularized latent space optimization. Nature Machine Intelligence, 4(10), 840-851.

## 2. Protein Interaction Level

* [Protein complex prediction with AlphaFold-Multimer](https://www.biorxiv.org/content/10.1101/2021.10.04.463034v2.full.pdf). *BioRxiv preprint*. Evans, R., O’Neill, M., Pritzel, A., Antropova, N., Senior, A., Green, T., ... & Hassabis, D. (2022). 
* [Predicting drug–protein interaction using quasi-visual question answering system](https://www.nature.com/articles/s42256-020-0152-y). *Nature Machine Intelligence*. Zheng, S., Li, Y., Chen, S., Xu, J., & Yang, Y. (2020). Predicting drug–protein interaction using quasi-visual question answering system. Nature Machine Intelligence, 2(2), 134-140.
* [A topology-based network tree for the prediction of protein–protein binding affinity changes following mutation](https://www.nature.com/articles/s42256-020-0149-6). *Nature Machine Intelligence*. Wang, M., Cang, Z., & Wei, G. W. (2020). A topology-based network tree for the prediction of protein–protein binding affinity changes following mutation. Nature Machine Intelligence, 2(2), 116-123.
* 




## 3. Cell Level

* [scBERT as a large-scale pretrained deep language model for cell type annotation of single-cell RNA-seq data](https://www.nature.com/articles/s42256-022-00534-z#Sec2). *Nature Machine Intelligence*. Yang, F., Wang, W., Wang, F., Fang, Y., Tang, D., Huang, J., ... & Yao, J. (2022). 

* [GEARS: Predicting transcriptional outcomes of novel multi-gene perturbations](https://www.biorxiv.org/content/10.1101/2022.07.12.499735v2). *BioRxiv preprint*. Roohani, Y., Huang, K., & Leskovec, J. (2022). 
* [Clustering single-cell RNA-seq data with a model-based deep learning approach](https://www.nature.com/articles/s42256-019-0037-0). *Nature Machine Intelligence*. Tian, T., Wan, J., Song, Q., & Wei, Z. (2019). Clustering single-cell RNA-seq data with a model-based deep learning approach. Nature Machine Intelligence, 1(4), 191-198.
* [An interpretable deep-learning architecture of capsule networks for identifying cell-type gene expression programs from single-cell RNA-sequencing data](https://www.nature.com/articles/s42256-020-00244-4). *Nature Machine Intelligence*. Wang, L., Nie, R., Yu, Z., Xin, R., Zheng, C., Zhang, Z., ... & Cai, J. (2020). An interpretable deep-learning architecture of capsule networks for identifying cell-type gene expression programs from single-cell RNA-sequencing data. Nature Machine Intelligence, 2(11), 693-703.
* [Iterative transfer learning with neural network for clustering and cell type classification in single-cell RNA-seq analysis](https://www.nature.com/articles/s42256-020-00233-7). *Nature Machine Intelligence*. Hu, J., Li, X., Hu, G., Lyu, Y., Susztak, K., & Li, M. (2020). Iterative transfer learning with neural network for clustering and cell type classification in single-cell RNA-seq analysis. Nature machine intelligence, 2(10), 607-618.
* [Simultaneous deep generative modelling and clustering of single-cell genomic data](https://www.nature.com/articles/s42256-021-00333-y). *Nature Machine Intelligence*. Liu, Q., Chen, S., Jiang, R., & Wong, W. H. (2021). Simultaneous deep generative modelling and clustering of single-cell genomic data. Nature machine intelligence, 3(6), 536-544.
* [A multi-use deep learning method for CITE-seq and single-cell RNA-seq data integration with cell surface protein prediction and imputation](https://www.nature.com/articles/s42256-022-00545-w). *Nature Machine Intelligence*. Lakkis, J., Schroeder, A., Su, K., Lee, M. Y., Bashore, A. C., Reilly, M. P., & Li, M. (2022). A multi-use deep learning method for CITE-seq and single-cell RNA-seq data integration with cell surface protein prediction and imputation. Nature Machine Intelligence, 1-13.
* [scBERT as a large-scale pretrained deep language model for cell type annotation of single-cell RNA-seq data](https://www.nature.com/articles/s42256-022-00534-z). *Nature Machine Intelligence*. Yang, F., Wang, W., Wang, F., Fang, Y., Tang, D., Huang, J., ... & Yao, J. (2022). scBERT as a large-scale pretrained deep language model for cell type annotation of single-cell RNA-seq data. Nature Machine Intelligence, 4(10), 852-866.
* [Contrastive learning enables rapid mapping to multimodal single-cell atlas of multimillion scale](https://www.nature.com/articles/s42256-022-00518-z). *Nature Machine Intelligence*. Yang, M., Yang, Y., Xie, C., Ni, M., Liu, J., Yang, H., ... & Wang, J. (2022). Contrastive learning enables rapid mapping to multimodal single-cell atlas of multimillion scale. Nature Machine Intelligence, 4(8), 696-709.
* [Interpreting the B-cell receptor repertoire with single-cell gene expression using Benisse](https://www.nature.com/articles/s42256-022-00492-6). *Nature Machine Intelligence*. Zhang, Z., Chang, W. Y., Wang, K., Yang, Y., Wang, X., Yao, C., ... & Wang, T. (2022). Interpreting the B-cell receptor repertoire with single-cell gene expression using Benisse. Nature Machine Intelligence, 1-9.
* [Simultaneous dimensionality reduction and integration for single-cell ATAC-seq data using deep learning](https://www.nature.com/articles/s42256-022-00443-1). *Nature Machine Intelligence*. Kopp, W., Akalin, A., & Ohler, U. (2022). Simultaneous dimensionality reduction and integration for single-cell ATAC-seq data using deep learning. Nature Machine Intelligence, 4(2), 162-168.
* [Cell type annotation of single-cell chromatin accessibility data via supervised Bayesian embedding](https://www.nature.com/articles/s42256-021-00432-w).*Nature Machine Intelligence*. Chen, X., Chen, S., Song, S., Gao, Z., Hou, L., Zhang, X., ... & Jiang, R. (2022). Cell type annotation of single-cell chromatin accessibility data via supervised Bayesian embedding. Nature Machine Intelligence, 4(2), 116-126.
* 

## 4. Others

* [Predicting disease-associated mutation of metal-binding sites in proteins using a deep learning approach](https://www.nature.com/articles/s42256-019-0119-z). *Nature Machine Intelligence*. Koohi-Moghadam, M., Wang, H., Wang, Y., Yang, X., Li, H., Wang, J., & Sun, H. (2019). Predicting disease-associated mutation of metal-binding sites in proteins using a deep learning approach. Nature Machine Intelligence, 1(12), 561-567.
* [Evaluation of deep learning in non-coding RNA classification](https://www.nature.com/articles/s42256-019-0051-2). *Nature Machine Intelligence*. Amin, N., McGrath, A., & Chen, Y. P. P. Evaluation of deep learning in non-coding RNA classification. Nat Mach Intell. 2019; 1 (5): 246–56. External Resources Crossref (DOI).
* [Feedback GAN for DNA optimizes protein functions](https://www.nature.com/articles/s42256-019-0017-4). *Nature Machine Intelligence*. Gupta, A., & Zou, J. (2019). Feedback GAN for DNA optimizes protein functions. Nature Machine Intelligence, 1(2), 105-111.
* [Gene set inference from single-cell sequencing data using a hybrid of matrix factorization and variational autoencoders](https://www.nature.com/articles/s42256-020-00269-9). *Nature Machine Intelligence*. Lukassen, S., Ten, F. W., Adam, L., Eils, R., & Conrad, C. (2020). Gene set inference from single-cell sequencing data using a hybrid of matrix factorization and variational autoencoders. Nature Machine Intelligence, 2(12), 800-809.
* [Elucidation of DNA methylation on N6-adenine with deep learning](https://www.nature.com/articles/s42256-020-0211-4). *Nature Machine Intelligence*. Tan, F., Tian, T., Hou, X., Yu, X., Gu, L., Mafra, F., ... & Hakonarson, H. (2020). Elucidation of DNA methylation on N6-adenine with deep learning. Nature Machine Intelligence, 2(8), 466-475.
* [Deep learning decodes the principles of differential gene expression](https://www.nature.com/articles/s42256-020-0201-6). *Nature Machine Intelligence*.
Tasaki, S., Gaiteri, C., Mostafavi, S., & Wang, Y. (2020). Deep learning decodes the principles of differential gene expression. Nature Machine Intelligence, 2(7), 376-386.
* [Gaussian embedding for large-scale gene set analysis](https://www.nature.com/articles/s42256-020-0193-2). *Nature Machine Intelligence*. Wang, S., Flynn, E. R., & Altman, R. B. (2020). Gaussian embedding for large-scale gene set analysis. Nature machine intelligence, 2(7), 387-395.
* [Deep learning incorporating biologically inspired neural dynamics and in-memory computing](https://www.nature.com/articles/s42256-020-0187-0) *Nature Machine Intelligence*. Woźniak, S., Pantazi, A., Bohnstingl, T., & Eleftheriou, E. (2020). Deep learning incorporating biologically inspired neural dynamics and in-memory computing. Nature Machine Intelligence, 2(6), 325-336.
* [A novel machine learning framework for automated biomedical relation extraction from large-scale literature repositories](https://www.nature.com/articles/s42256-020-0189-y). *Nature Machine Intelligence*. Hong, L., Lin, J., Li, S., Wan, F., Yang, H., Jiang, T., ... & Zeng, J. (2020). A novel machine learning framework for automated biomedical relation extraction from large-scale literature repositories. Nature Machine Intelligence, 2(6), 347-355.
* [A deep learning method for recovering missing signals in transcriptome-wide RNA structure profiles from probing experiments](https://www.nature.com/articles/s42256-021-00412-0). *Nature Machine Intelligence*. Gong, J., Xu, K., Ma, Z., Lu, Z. J., & Zhang, Q. C. (2021). A deep learning method for recovering missing signals in transcriptome-wide RNA structure profiles from probing experiments. Nature Machine Intelligence, 3(11), 995-1006.
* [Integration of multiomics data with graph convolutional networks to identify new cancer genes and their associated molecular mechanisms](https://www.nature.com/articles/s42256-021-00325-y). *Nature Machine Intelligence*. Schulte-Sasse, R., Budach, S., Hnisz, D., & Marsico, A. (2021). Integration of multiomics data with graph convolutional networks to identify new cancer genes and their associated molecular mechanisms. Nature Machine Intelligence, 3(6), 513-526.
* [An automated framework for efficiently designing deep convolutional neural networks in genomics](https://www.nature.com/articles/s42256-021-00316-z). *Nature Machine Intelligence*. Zhang, Z., Park, C. Y., Theesfeld, C. L., & Troyanskaya, O. G. (2021). An automated framework for efficiently designing deep convolutional neural networks in genomics. Nature Machine Intelligence, 3(5), 392-400.
* [Improving representations of genomic sequence motifs in convolutional networks with exponential activations](https://www.nature.com/articles/s42256-020-00291-x). *Nature Machine Intelligence*. Repecka, D., Jauniskis, V., Karpus, L., Rembeza, E., Rokaitis, I., Zrimec, J., ... & Zelezniak, A. (2021). Expanding functional protein sequence spaces using generative adversarial networks. Nature Machine Intelligence, 3(4), 324-333.
* [Deep neural networks identify sequence context features predictive of transcription factor binding](https://www.nature.com/articles/s42256-020-00282-y). *Nature Machine Intelligence*. Zheng, A., Lamkin, M., Zhao, H., Wu, C., Su, H., & Gymrek, M. (2021). Deep neural networks identify sequence context features predictive of transcription factor binding. Nature machine intelligence, 3(2), 172-180.
* [Deep neural networks with controlled variable selection for the identification of putative causal genetic variants](https://www.nature.com/articles/s42256-022-00525-0). *Nature Machine Intelligence*. Kassani, P. H., Lu, F., Le Guen, Y., Belloy, M. E., & He, Z. (2022). Deep neural networks with controlled variable selection for the identification of putative causal genetic variants. Nature Machine Intelligence, 4(9), 761-771.
* [Molecular convolutional neural networks with DNA regulatory circuits](https://www.nature.com/articles/s42256-022-00502-7). *Nature Machine Intelligence*. Xiong, X., Zhu, T., Zhu, Y., Cao, M., Xiao, J., Li, L., ... & Pei, H. (2022). Molecular convolutional neural networks with DNA regulatory circuits. Nature Machine Intelligence, 4(7), 625-635.
* [Interpreting neural networks for biological sequences by learning stochastic masks](https://www.nature.com/articles/s42256-021-00428-6).*Nature Machine Intelligence*. Linder, J., La Fleur, A., Chen, Z., Ljubetič, A., Baker, D., Kannan, S., & Seelig, G. (2022). Interpreting neural networks for biological sequences by learning stochastic masks. Nature Machine Intelligence, 4(1), 41-54.
* [Hierarchical deep reinforcement learning reveals a modular mechanism of cell movement](https://www.nature.com/articles/s42256-021-00431-x). *Nature Machine Intelligence*. Wang, Z., Xu, Y., Wang, D., Yang, J., & Bao, Z. (2022). Hierarchical deep reinforcement learning reveals a modular mechanism of cell movement. Nature Machine Intelligence, 4(1), 73-83.
